utxo/utxo.go

```diff
@@ -1409,7 +1409,7 @@ func (uv *UtxoVM) verifyTxRWSets(tx *pb.Transaction) (bool, error) {
 		if err != nil {
 			// FIXME zq @icexin: need to return contract not found
 			uv.xlog.Error("verifyTxRWSets NewContext error", "err", err, "contractName", tmpReq.GetContractName())
-			if i < len(reservedRequests) && err.Error() == "leveldb: not found" {
+			if i < len(reservedRequests) && (err.Error() == "leveldb: not found" || strings.HasSuffix(err.Error(), "not found")) {
```

Comment:
When the reserved contracts are not installed, It will return Key not found.

Reply:
> When the reserved contracts are not installed, It will return Key not found.

Return an error which has an explicitly type will be better.

Reply:
Yes, model cache will support it.